lrz_validator/CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ set(LIB_SRC_FILES ${SRC_FILES})
 list(FILTER LIB_SRC_FILES EXCLUDE REGEX "main.cpp")
 
 add_executable(${PROJECT_NAME} ${HDR_FILES} "main.cpp")
+target_include_directories(${PROJECT_NAME} PRIVATE ${THIRDPARTY_DIRECTORY}/gfxreconstruct/framework)
```

Comment:
sorry why is this needed? is this the same as line 47?

Reply:
Removed.